rllib/tuned_examples/sac/humanoid_sac.py

```diff
@@ -0,0 +1,69 @@
+"""This is WIP.
+
+On a single-GPU machine, with the --num-gpus=1 command line option, this example should
+learn a episode return of >1000 in ~10h, which is still very basic, but does somewhat
+prove SAC's capabilities. Some more hyperparameter fine tuning, longer runs, and
+more scale (`--num-gpus > 1` and `--num-env-runners > 0`) should help push this up.
+"""
+
+from torch import nn
+
+from ray.rllib.algorithms.sac.sac import SACConfig
+from ray.rllib.utils.test_utils import add_rllib_example_script_args
+
+parser = add_rllib_example_script_args(
+    default_timesteps=1000000,
+    default_reward=12000.0,
+    default_iters=2000,
+)
+parser.set_defaults(enable_new_api_stack=True)
+# Use `parser` to add your own custom command line options to this script
+# and (if needed) use their values to set up `config` below.
+args = parser.parse_args()
+
+
+config = (
+    SACConfig()
+    .api_stack(
+        enable_rl_module_and_learner=True,
+        enable_env_runner_and_connector_v2=True,
+    )
+    .environment("Humanoid-v4")
+    .training(
+        initial_alpha=1.001,
+        lr=0.00005,
```

Comment:
I wonder how this can run with a learning rate not `None`? The new PR is merged and requires `lr=None` in the new stack. Instead `actor_lr`, `critic_lr`, and `alpha_lr`  can be used.

Reply:
True, let me double check ...

Reply:
Great catch! I must have still run it on the old master (before we merged that PR).

Fixed.